update.go

```diff
@@ -189,7 +191,7 @@ func checkImageDate(profile *Profile, checkData ImageCheckData, force bool) []Im
 		imageCandidates = append(imageCandidates, ImageDef{Image: profile.ImageAMD64, Platform: "linux/amd64"})
 		imageCandidates = append(imageCandidates, ImageDef{Image: profile.ImageARM64, Platform: "linux/arm64"})
 	case profile.Image != "":
-		imageCandidates = append(imageCandidates, ImageDef{Image: profile.Image, Platform: profile.Arch})
+		imageCandidates = append(imageCandidates, ImageDef{Image: profile.Image, Platform: "linux/" + profile.Arch})
```

Comment:
This was a bug I noticed along the way. (We don't have any single-arch profiles in use, so it's probably never been hit.)

Reply:
I saw that and thought it was interesting...